hrm_oo.py

```diff
@@ -0,0 +1,90 @@
+from readHR import EcgReader
+from hrmcalcs2oo import hrmcalcs
+from hrmtb import TachyBrady
+
+
+class HrmOutput:
+
+    def __init__(self, file='Test_ECG.csv', start_min=3, end_min=5,
+                 brady_limit=60, tachy_limit=100):
+        """
+
+        :param file: The file to make heart rate calculations for
+
+        :param start_min: User-inputted value for the "starting minute" of
+        average heart rate calculations, in minutes.
+
+        :param end_min: User-inputted value for the "ending minute" of average
+        heart rate calculations, in minutes.
+
+        :param brady_limit: The threshold value for bradycardia, with the
+        default set at 60 bpm
+
+        :param tachy_limit: The threshold value for tachycardia, with the
+        default set at 100 bpm
+        """
+        self.file = file
+        self.start_min = start_min
+        self.end_min = end_min
+        self.brady_limit = brady_limit
+        self.tachy_limit = tachy_limit
+        self.valid_file = False
+        self.peak_vector = []
+        self.instant_hr = []
+        self.average_hr = None
+        self.tachy = []
+        self.brady = []
+        if self.valid_file:
```

Comment:
Will `self.valid_file` always be `False` when the constructor is executed, since it is set to False on L31?

Reply:
Whoops, yep you're right. I meant for this to be one line down (after extractvals, which would change it to true)